integrations/to.etc.domui.selenium/src/main/java/to/etc/domui/webdriver/core/base/TabPO.java

```diff
@@ -0,0 +1,88 @@
+package to.etc.domui.webdriver.core.base;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openqa.selenium.By;
+import to.etc.domui.webdriver.core.WebDriverConnector;
+import to.etc.util.Pair;
+
+import java.util.Map;
+
+import static java.util.stream.Collectors.toMap;
+
+@NonNullByDefault
+public class TabPO extends BasePO {
+	@Nullable
+	private String m_current;
+
+	private String m_id;
+
+	private Map<String, ? extends BasePO> m_tabs;
+
+	/**
+	 * @param wd web driver
+	 * @param testId of the Tab
+	 * @param tabs list of configured tabs.
+	 */
+	public TabPO(WebDriverConnector wd, String testId, Map<String,? extends BasePO> tabs) {
```

Comment:
If you want to define the tab panels as PO's themselves it's better to do that with an add() method instead of via the constructor, I think.